pkg/dns/vips/persistence_test.go

```diff
@@ -123,6 +127,18 @@ var _ = Describe("Meshed Persistence", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(actual).To(Equal(expected))
 		})
+
+		It("should return vips for mesh in new format", func() {
```

Comment:
how do we know it's a new format? looks pretty much the same as `should return vips for mesh`. I diffed this with `should return vips for mesh` and it's almost the same except redis is missing and `origin="service"`?

Reply:
json with new format is above: https://github.com/kumahq/kuma/pull/7088/files#diff-85e4a53f8349e8a7d876f9fe09fa9fc81480ebfba25853f14bf0d67823dfd10dR112